mlflow-server-oc-deployment-template.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
```

Comment:
Then name of the file should be `mlflow-server-template.yaml` and move it to a subdirectory called `openshift`, so it's consistent with our other repos, see https://github.com/AICoE/prometheus-flatliner